### DISCUSSION TITLE
Default language

### DISCUSSION THREAD

**Discussion by tobiasBora** [minad/jinx#153] [Q&A] [answered]
Thanks for this great software. I’m just surprised, I can’t find any way to select a different language by default. I guess I could call `(jinx-languages "en")` in the hook of each opened file, but it will always ask to change the `Local Variables` field and never take into account variables like:
```
% Local Variables:
% jinx-languages: "en"
% End:
```
Am I missing something?

**Comment by tobiasBora** [comment]
Oh, seems like setting:
```
(use-package jinx
  :hook (emacs-startup . (lambda ()
                           (global-jinx-mode)
                           (setq jinx-languages "en")))
  :bind (("M-$" . jinx-correct)
         ("C-M-$" . jinx-languages)))
```
does the job. It might help to make this clear from the documentation?

**Reply by minad** [reply, marked as answer]
Setting `jinx-languages` to a default value should work (before enabling Jinx). By default the value is taken from the locale or the LANG environment variable.
